package.json

```diff
@@ -44,7 +49,9 @@
   "scripts": {
     "test": "grunt test",
     "coveralls": "cross-env NODE_ENV=test nyc mocha && cat ./coverage/lcov.info | coveralls",
-    "prepare": "grunt compile"
+    "prepare": "grunt compile",
+    "lint": "node_modules/.bin/eslint ./src ./test",
```

Comment:
Nitpick: `node_modules/.bin/` part is superfluous; `npm` automatically includes binaries into the path.

Reply:
Sure - my fault, solved :)